core/src/main/java/tc/oc/pgm/commands/AdminCommands.java

```diff
@@ -115,13 +124,61 @@ public static void setNext(
                   .translate("command.admin.cancelRestart.restartUnqueued", sender));
     }
 
-    sender.sendMessage(
-        ChatColor.DARK_PURPLE
-            + AllTranslations.get()
-                .translate(
-                    "command.admin.set.success",
-                    sender,
-                    ChatColor.GOLD + map.getName() + ChatColor.DARK_PURPLE));
+    Component mapName = new PersonalizedText(map.getName()).color(ChatColor.DARK_PURPLE);
+    Component successful =
+        new PersonalizedTranslatable(
+                "command.admin.set.success",
+                ModerationCommands.formatStaffName(sender, match),
+                mapName)
+            .getPersonalizedText()
+            .color(ChatColor.GRAY);
+    ChatDispatcher.broadcastAdminChatMessage(successful, match);
+  }
+
+  @Command(
+      aliases = {"setpool", "setrot"},
+      desc = "Set a custom map pool or revert to dynamic",
+      usage = "[pool name] -r (revert to dynamic)",
+      flags = "r",
+      perms = Permissions.SETNEXT)
+  public static void setPool(
+      CommandSender sender,
+      Match match,
+      MapOrder mapOrder,
+      @Nullable String poolName,
+      @Switch('r') boolean reset)
+      throws CommandException {
+    if (!match.getCountdown().getAll(CycleCountdown.class).isEmpty()) {
+      sender.sendMessage(
+          new PersonalizedTranslatable("command.admin.setPool.activeCycle")
+              .getPersonalizedText()
+              .color(ChatColor.RED));
+      return;
+    }
+    MapPoolManager mapPoolManager = MapPoolCommands.getMapPoolManager(sender, mapOrder);
+    MapPool newPool =
+        reset
+            ? mapPoolManager.getAppropriateDynamicPool(match).orElse(null)
+            : mapPoolManager.getMapPoolByName(poolName);
+
+    if (newPool == null) {
+      Component noPoolError =
+          new PersonalizedTranslatable("command.pools.noPoolMatch")
```

Comment:
Im guessing this will be displayed if someone does `/setpool -r` and there is no dynamic pools present? I think there should be a more intuitive response if so (e.g "No dynamic pools loaded") 

This could maybe be located in `getAppropriateDynamicPool`?